prow/jobs/test-infra/check_config_files.yaml

```diff
@@ -0,0 +1,17 @@
+presubmits:
```

Comment:
File name should be consistent with the other jobs

Reply:
Do we treat the whole test-infra as a single component, and I should merge the job into `prow/jobs/test-infra/test-infra.jobs.yaml`, or do we consider building the docker images a different component than checking config files and want to have it in different files?